crates/sui-benchmark/src/drivers/bench_driver.rs

```diff
@@ -435,7 +435,7 @@ impl Driver<BenchmarkStats> for BenchDriver {
                                     num_success += 1;
                                     num_in_flight -= 1;
                                     free_pool.push(new_payload);
-                                    latency_histogram.record(latency.as_millis().try_into().unwrap()).unwrap();
+                                    latency_histogram.saturating_record(latency.as_millis().try_into().unwrap());
```

Comment:
Is there some reason that you increased the max value to `120_000` and changed this to `saturating_record`? Looks like it catches values over max without warning, so we can keep the old max unless there is some reason 120s was chosen.

Reply:
The 120s change is not necessary. But I think the `saturating_record` should be used as a workaround. Otherwise, I see the error during ramp up phase of benchmark client. Maybe after https://github.com/MystenLabs/sui/issues/6984 this will be no longer an issue. And if we want to crash on high latency writes, maybe it should be done deliberately.